lab-beeksma/model/workout.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const debug = require('debug')('app:note');
+const storage = require('../lib/storage');
+const uuidv4 = require('uuid/v4');
+const createError = require('http-errors');
+
+const Workout = module.exports = function(exercise, weight, set, rep){
+  this.id = uuidv4();
+  this.exercise = exercise;
+  this.weight = weight;
+  this.set = set;
+  this.rep = rep;
+};
+
+Workout.createWorkout = function(body){
+  if(!body.exercise) {
+    return Promise.reject(createError(400, 'body is missing'));
```

Comment:
"body is missing" doesn't seem to align with the `if` condition? What if `body` really is missing?